Manifests/ManagedPreferencesApplications/io.tailscale.ipn.macos.plist

```diff
@@ -0,0 +1,434 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<plist version="1.0">
+<dict>
+	<key>pfm_app_url</key>
+	<string>https://apps.apple.com/app/tailscale/id1475387142</string>
+	<key>pfm_description</key>
+	<string>Configure settings for the Mac App Store variant of the Tailscale client.</string>
```

Comment:
Is this really only for the App store version and not also the PKG version of Tailscale?

Reply:
disregard, I forgot different domains